src/main/java/com/synopsys/integration/alert/channel/azure/boards/service/AzureBoardsIssueHandler.java

```diff
@@ -0,0 +1,176 @@
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.synopsys.integration.alert.channel.azure.boards.model.AzureBoardsIssueConfig;
+import com.synopsys.integration.alert.common.channel.issuetracker.config.IssueConfig;
+import com.synopsys.integration.alert.common.channel.issuetracker.enumeration.IssueOperation;
+import com.synopsys.integration.alert.common.channel.issuetracker.message.AlertIssueOrigin;
+import com.synopsys.integration.alert.common.channel.issuetracker.message.IssueContentLengthValidator;
+import com.synopsys.integration.alert.common.channel.issuetracker.message.IssueContentModel;
+import com.synopsys.integration.alert.common.channel.issuetracker.message.IssueSearchProperties;
+import com.synopsys.integration.alert.common.channel.issuetracker.message.IssueTrackerIssueResponseModel;
+import com.synopsys.integration.alert.common.channel.issuetracker.message.IssueTrackerRequest;
+import com.synopsys.integration.alert.common.channel.issuetracker.service.IssueHandler;
+import com.synopsys.integration.azure.boards.common.service.workitem.AzureWorkItemService;
+import com.synopsys.integration.azure.boards.common.service.workitem.request.WorkItemElementOperation;
+import com.synopsys.integration.azure.boards.common.service.workitem.request.WorkItemElementOperationModel;
+import com.synopsys.integration.azure.boards.common.service.workitem.request.WorkItemRequest;
+import com.synopsys.integration.azure.boards.common.service.workitem.response.WorkItemResponseFields;
+import com.synopsys.integration.azure.boards.common.service.workitem.response.WorkItemResponseModel;
+import com.synopsys.integration.azure.boards.common.util.AzureFieldDefinition;
+import com.synopsys.integration.exception.IntegrationException;
+import com.synopsys.integration.rest.exception.IntegrationRestException;
+
+public class AzureBoardsIssueHandler extends IssueHandler<WorkItemResponseModel> {
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    private static final IssueContentLengthValidator CONTENT_LENGTH_VALIDATOR = new IssueContentLengthValidator(
+        AzureBoardsMessageParser.TITLE_SIZE_LIMIT,
+        AzureBoardsMessageParser.MESSAGE_SIZE_LIMIT,
+        AzureBoardsMessageParser.MESSAGE_SIZE_LIMIT
+    );
+
+    private final AzureBoardsMessageParser azureBoardsMessageParser;
+    private final AzureWorkItemService azureWorkItemService;
+
+    public AzureBoardsIssueHandler(AzureBoardsMessageParser azureBoardsMessageParser, AzureWorkItemService azureWorkItemService) {
+        super(CONTENT_LENGTH_VALIDATOR);
+        this.azureBoardsMessageParser = azureBoardsMessageParser;
+        this.azureWorkItemService = azureWorkItemService;
+    }
+
+    @Override
+    protected Optional<WorkItemResponseModel> createIssue(IssueConfig issueConfig, IssueTrackerRequest request) throws IntegrationException {
+        AzureBoardsIssueConfig azureBoardsIssueConfig = (AzureBoardsIssueConfig) issueConfig;
```

Comment:
Would generics work well here? Or do we prefer casting manually?

Reply:
Not in this case. Azure has an edge case where it has an extra field in `IssueConfig`.

Reply:
Turns out, this was not actually an edge case. When we actually need the extra field, we have it. Just an overly cautious add on my part. This class has been deleted.